pkg/aws/helpers.go

```diff
@@ -258,6 +258,9 @@ func UserTagValidator(input interface{}) error {
 			if len(tag) != 2 {
 				return fmt.Errorf("invalid tag format. Expected tag format: 'key:value'")
 			}
+			if tag[1] == "" {
+				return fmt.Errorf("invalid tag format, Tags must have values. Expected tag format: 'key:value'")
+			}
```

Comment:
This way totally works, I just wonder if it would instead be better to just update the UserTagValueRE expression. It is `^[\pL\pZ\pN_.:/=+\-@]{0,256}$`, I believe `^[\pL\pZ\pN_.:/=+\-@]{1,256}$` would cover what we need, which is consisten with the way that the UserTagKeyRE is validated.